docs/source/architecture/sub_stakes.rst

```diff
@@ -0,0 +1,276 @@
+
+**Example:**
+
+    A staker has few sub-stakes and calls ``mint()``. Assume that thus far the 1st and 2nd sub-stakes will produce 50 tokens and 20 tokens respectively in rewards:
+		- Before calling: 
+			* 1st sub-stake = 400 tokens for 8 periods
+			* 2nd sub-stake = 200 tokens for 2 periods
+			* 3rd sub-stake = 100 tokens locked starting from the next period and locked for 5 periods
+			* 100 tokens in an unlocked state
+		- After calling, if re-staking is disabled:  
+			* 1st sub-stake = 400 tokens for 8 periods
+			* 2nd sub-stake = 200 tokens for 2 periods
+			* 3rd sub-stake = 100 tokens locked starting from the next period and locked for 5 periods
+			* 170 tokens in an unlocked state
+		- After calling, if re-staking is enabled: 
+			* 1st sub-stake = 450 tokens for 8 periods
+			* 2nd sub-stake = 220 tokens for 2 periods
+			* 3rd sub-stake = 100 tokens locked starting from the next period and locked for 5 periods
+			* 100 tokens in an unlocked state
+
+    .. aafig::
+        :proportional:
+        :textual:
+
+                             Before             
+
+            stake
+            ^
+            |
+         700|   +-------+
+            |   |  3rd  |
+         600+---+-------+
+         500|           +-----------+
+            |    2nd    |    3rd    |
+         400+-----------+-----------+-----------+
+            |                                   |
+            |                1st                |   period
+            +---+---+---+---+---+---+---+---+---+---->    
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
+			
+			
+			
+			
+                         After, re–staking is enabled             
+
+            stake
+            ^
+            |
+         770|   +-------+
+            |   |  3rd  |
+         670+---+-------+
+            |           |
+         550|    2nd    +-----------+
+            |           |    3rd    |
+         450+-----------+-----------+-----------+
+            |                                   |
+            |                1st                |
+            |                                   |   period
+            +---+---+---+---+---+---+---+---+---+---->    
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
+			
+
+
+Winding down
+^^^^^^^^^^^^
+*Used in methods* : ``confirmActivity()``
+
+A disabled "winding down" parameter (default) guarantees that the worker must call ``confirmActivity()`` at least N times after the parameter is enabled to unlock the sub-stake, where N is the locking duration of sub-stake. When disabled, the unlock date for each sub-stakes shifts forward by 1 period after each period i.e. the duration continues to remain the same until the parameter is enabled. Once the "winding down" parameter is enabled, each call to ``confirmActivity()`` (no more than once in a period) leads to a reduction of locking duration for each sub-stake, and the unlock date no longer changes.
```

Comment:
Is it more intuitive to explain an enabled "winding down" parameter first?

Something like:
```suggestion
An enabled "winding down" parameter means that each call to ``confirmActivity()`` (no more than once in a period) leads to a reduction of locking duration for each sub-stake. It guarantees that the sub-stake will unlock after the worker calls ``confirm-activity()`` at least N times (no more than once in a period), where N is the locking duration of sub-stake. When disabled (default), the unlock date for each sub-stakes shifts forward by 1 period after each period i.e. the duration continues to remain the same until the parameter is enabled.
```

Reply:
I guess it will be better but something wrong with this particular suggestion (confused by guranties part)

Reply:
```suggestion
An enabled "winding down" parameter means that each call to ``confirmActivity()`` (no more than once in a period) leads to a reduction of locking duration for each sub-stake. In other words, the sub-stake will unlock after the worker calls ``confirm-activity()`` at least N times (no more than once in a period), where N is the locking duration of sub-stake. When disabled (default), the unlock date for each sub-stakes shifts forward by 1 period after each period. In other words, the duration continues to remain the same until the "winding down" parameter is enabled.```

Reply:
added!
Also if you could add somewhere that when winding down enabled - unlock date remains the same